addons/spreadsheet_dashboard/models/spreadsheet_dashboard_group.py

```diff
@@ -19,3 +19,28 @@ def _unlink_except_spreadsheet_data(self):
             external_id = external_ids[group.id]
             if external_id and not external_id.startswith('__export__'):
                 raise UserError(_("You cannot delete %s as it is used in another module.", group.name))
+
+    @api.model
```

Comment:
```suggestion
    @api.model
    @api.readonly
```
could be marked as readonly :wink: